src/components/governance/PageProposal.vue

```diff
@@ -150,85 +154,99 @@ export default {
+    endDate() {
+      return moment(this.proposal.voting_end_time).format("MMMM Do YYYY, HH:mm")
     },
-    endDate({ proposal } = this) {
-      return moment(proposal.voting_end_time).format("MMMM Do YYYY, HH:mm")
-    },
-    displayEndDate({ proposal, governanceParameters } = this) {
+    displayEndDate({ governanceParameters } = this) {
       if (
-        proposal.proposal_status !== "DepositPeriod" &&
-        proposal.total_deposit[0].amount >=
+        this.proposal.status !== "DepositPeriod" &&
+        this.proposal.total_deposit[0].amount >=
           Number(governanceParameters.parameters.deposit.min_deposit[0].amount)
       ) {
         return true
       } else {
         return false
       }
     },
-    votingStartedAgo({ proposal } = this) {
-      return moment(proposal.voting_start_time).format("MMMM Do YYYY, HH:mm")
+    votingStartedAgo() {
+      return moment(this.proposal.voting_start_time).format(
+        "MMMM Do YYYY, HH:mm"
+      )
     },
-    depositEndsIn({ proposal } = this) {
-      return moment(new Date(proposal.deposit_end_time)).fromNow()
+    depositEndsIn() {
+      return moment(new Date(this.proposal.deposit_end_time)).fromNow()
     },
-    totalVotes({ tally: { yes, no, no_with_veto, abstain } } = this) {
-      return BigNumber(yes)
-        .plus(no)
-        .plus(no_with_veto)
-        .plus(abstain)
+    totalVotes({ tally: { final_tally_yes, final_tally_no, final_tally_no_with_veto, final_tally_abstain } } = this) {
+      console.log()
+      return BigNumber(final_tally_yes)
+        .plus(final_tally_no)
+        .plus(final_tally_no_with_veto)
+        .plus(final_tally_abstain)
         .toNumber()
     },
     totalVotePercentage({ pool, totalVotes }) {
       const totalPossibleVotes = pool.pool.bonded_tokens
       return num.percentInt(totalVotes / totalPossibleVotes)
     },
     yesPercentage({ tally, totalVotes } = this) {
-      return num.percentInt(totalVotes === 0 ? 0 : tally.yes / totalVotes)
+      return num.percentInt(totalVotes === 0 ? 0 : tally.final_tally_yes / totalVotes)
     },
     noPercentage({ tally, totalVotes } = this) {
-      return num.percentInt(totalVotes === 0 ? 0 : tally.no / totalVotes)
+      return num.percentInt(totalVotes === 0 ? 0 : tally.final_tally_no / totalVotes)
     },
     noWithVetoPercentage({ tally, totalVotes } = this) {
       return num.percentInt(
-        totalVotes === 0 ? 0 : tally.no_with_veto / totalVotes
+        totalVotes === 0 ? 0 : tally.final_tally_no_with_veto / totalVotes
       )
     },
     abstainPercentage({ tally, totalVotes } = this) {
-      return num.percentInt(totalVotes === 0 ? 0 : tally.abstain / totalVotes)
+      return num.percentInt(totalVotes === 0 ? 0 : tally.final_tally_abstain / totalVotes)
     },
-    tally({ proposals, proposalId } = this) {
-      const { yes, no, abstain, no_with_veto } =
-        proposals.tallies[proposalId] || {}
+    tally() {
+      const {
```

Comment:
I guess we need to rework this, once the data schema is adjusted to standardize current tally and final tally.